partner_industry_secondary/views/res_partner_view.xml

```diff
@@ -70,15 +82,11 @@
             </filter>
         </field>
     </record>
-    <!-- Views activated for group_use_partner_industry_for_person -->
```

Comment:
Any reason to comment all these views?

Reply:
No reason, I just removed the commented views.